linters/javascript/.eslintrc

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": "airbnb",
+  "parser": "babel-eslint",
+  "rules": {
+    "consistent-return": 0,
+    "id-length": [2, {"exceptions": ["_"]}],
+    "new-cap": [2, {
+      "capIsNewExceptions": ["CSSModules"]
+    }],
+    "react/no-multi-comp": [2, { "ignoreStateless": true }]
+  }
+}
```

Comment:
according to [this section](http://eslint.org/docs/user-guide/configuring.html#configuration-file-formats) the `eslintrc` filename is deprecated.`eslintrc.json` is supported and doesn't require any changes to the contents